content/influxdb/v2.0/install.md

```diff
@@ -545,6 +546,25 @@ Always inspect YAML manifests before running `kubectl apply -f <url>`!
 
 {{% /tab-content %}}
 <!--------------------------------- END kubernetes ---------------------------->
+<!--------------------------------- BEGIN Rasberry Pi ------------------------->
+{{% tab-content %}}
+
+To run InfluxDB on Raspberry Pi, you will need:
+
+- a Raspberry Pi 4+ or 400
+- a 64-bit operating system.
+  We recommend installing a [64-bit version of Ubuntu](https://ubuntu.com/download/raspberry-pi)
+  of Ubuntu Desktop or Ubuntu Server compatible with 64-bit Raspberry Pi.
+
+Follow the [Linux installation instructions](/influxdb/v2.0/install/?t=Linux) to install InfluxDB on a Raspberry Pi.
+
+#### Support for 32-bit systems
+
+If you have a 32-bit Raspberry Pi, [use Telegraf](/{{< latest "telegraf" >}}/)
```

Comment:
I think we need a pointer to the a few things here:
Telegraf as the agent
and which plugins to configure.  --  or...before you get to the configure part suggest that this is covered as part of the InfluxDB Template for Rasp Pi??